### ISSUE TITLE
'class_weight' does not work with 'lregression'

### ISSUE THREAD

**Issue by ArdalanM** [beniz/deepdetect#215]
Calling dd with ```class_weight``` and `template:lregression` result in error:

```
ERROR - 17:52:31 - service pydd_lr_2016-11-25-17-52-30-082915 training status call failed

ERROR - 17:52:31 - {"code":500,"msg":"InternalError","dd_code":1007,"dd_msg":"src/caffe/layers/softmax_infogain_loss_layer.cpp:72 / Check failed (custom): (infogain->count()) == (num_labels_*num_labels_)"}
```

**Comment by beniz** [comment]
Could you try PR #218 ? This should also fix #216. Thanks.

**Comment by ArdalanM** [comment]
PR #218 fixed my problem, thanks !
